org.versotile.verso.yml

```diff
@@ -36,7 +36,9 @@ modules:
       # Build Verso release profile
       - cargo --offline fetch --manifest-path Cargo.toml
       - cargo --offline build --release --features flatpak
+      - cargo --offline build --release --package verso
       # Install Verso
+      - install -Dm755 ./target/release/versoview -t /app/bin/
       - install -Dm755 ./target/release/verso -t /app/bin/
```

Comment:
Do we need this line then?

Reply:
Do you mean we just change the flatpak's entry point to `versoview`?

Reply:
No I mean only `versoview` is used here right? So I'm think maybe there just need a install command.

Reply:
I'm not entirely sure what do you mean, do you want this flatpak to run `versoview` or `verso`? If `verso` then this is definitely needed right? If not we will change the command to `versoview` and don't build `verso` at all, so the entry point will be `versoview` right?

Reply:
ah you are right. I misunderstood the command earlier.